Parsers/ASimAuthentication/Parsers/ASimAuthentication.yaml

```diff
@@ -11,7 +11,7 @@ References:
 - Title: ASIM Authentication Schema
   Link: https://aka.ms/ASimAuthenticationDoc
 - Title: ASIM
-  Link: https:/aka.ms/AboutASIM
+  Link: https://aka.ms/AboutASIM
 Description: |
       This Query creates an empty table of the authentication schema.
```

Comment:
To be consistent use "This function returns an empty ASIM Authentication schema"